instance/tests/models/test_log_entry.py

```diff
@@ -67,14 +67,35 @@ def test_log_text(self):
         with freeze_time("2015-08-05 18:07:06"):
             server.logger.critical('Line #7, exception')
 
-        self.assertEqual(instance.log_text, ('''\
-2015-08-05 18:07:00 |     INFO | instance.models.instance  | instance=my.instance | Line #1, on instance
-2015-08-05 18:07:01 |     INFO | instance.models.server    | instance=my.instance,server=vm1_id | Line #2, on server
-2015-08-05 18:07:03 |     INFO | instance.models.instance  | instance=my.instance | Line #4, on instance
-2015-08-05 18:07:04 |  WARNING | instance.models.instance  | instance=my.instance | Line #5, on instance (warn)
-2015-08-05 18:07:05 |     INFO | instance.models.server    | instance=my.instance,server=vm1_id | Line #6, on server
-2015-08-05 18:07:06 | CRITICAL | instance.models.server    | instance=my.instance,server=vm1_id | Line #7, exception
-'''))
+        entries = instance.log_entries
+        self.assertEqual(entries[0].level, "INFO")
+        self.assertEqual(entries[0].created.strftime("%Y-%m-%d %H:%M:%S"), "2015-08-05 18:07:00")
+        self.assertEqual(entries[0].text,
+                         "instance.models.instance  | instance=my.instance | Line #1, on instance")
+
+        self.assertEqual(entries[1].level, "INFO")
+        self.assertEqual(entries[1].created.strftime("%Y-%m-%d %H:%M:%S"), "2015-08-05 18:07:01")
+        self.assertEqual(entries[1].text,
+                         "instance.models.server    | instance=my.instance,server=vm1_id | Line #2, on server")
+        self.assertEqual(entries[2].level, "INFO")
+        self.assertEqual(entries[2].created.strftime("%Y-%m-%d %H:%M:%S"), "2015-08-05 18:07:03")
+        self.assertEqual(entries[2].text,
+                         "instance.models.instance  | instance=my.instance | Line #4, on instance")
+
+        self.assertEqual(entries[3].level, "WARNING")
+        self.assertEqual(entries[3].created.strftime("%Y-%m-%d %H:%M:%S"), "2015-08-05 18:07:04")
+        self.assertEqual(entries[3].text,
+                         "instance.models.instance  | instance=my.instance | Line #5, on instance (warn)")
+
+        self.assertEqual(entries[4].level, "INFO")
+        self.assertEqual(entries[4].created.strftime("%Y-%m-%d %H:%M:%S"), "2015-08-05 18:07:05")
+        self.assertEqual(entries[4].text,
+                         "instance.models.server    | instance=my.instance,server=vm1_id | Line #6, on server")
+
+        self.assertEqual(entries[5].level, "CRITICAL")
+        self.assertEqual(entries[5].created.strftime("%Y-%m-%d %H:%M:%S"), "2015-08-05 18:07:06")
+        self.assertEqual(entries[5].text,
+                         "instance.models.server    | instance=my.instance,server=vm1_id | Line #7, exception")
```

Comment:
This is a bit unwieldy and hard to follow.  Maybe you could factor out a helper function to check a single log entry?